### PR TITLE
Database - New Tricks and Fixes

### PR DESCRIPTION
**Pull request by toasterparty**
These are all things brought up by the community in the discord server. I'll let the `.txt` diff speak for itself.

### CODE REVIEW

File: randovania/data/json_data/prime1/Phendrana Drifts.txt [hunk 1]
```diff
@@ -59,6 +59,10 @@ Asset id: 4146616697
   * Normal Door to Ruins Entryway/Door to Phendrana Shorelines
   > Door to Shoreline Entrance
       Trivial
+  > Door to Temple Entryway
+      All of the following:
+          Combat/Scan Dash (Intermediate)
+          Scan Visor or X-Ray Visor
```
Comment (dyceron) [inline randovania/data/json_data/prime1/Phendrana Drifts.txt:65]: Cant this be done in any visor? If so, could be an invisible objects intermediate or something added on to this

Comment (toasterparty) [reply randovania/data/json_data/prime1/Phendrana Drifts.txt:65]: The only viable scan point is the flicker bats. They cannot be locked onto with combat visor, but they can be locked onto with x-ray.

**Comment by dyceron** [comment]
Make the appropriate changelog entries as well.

**Comment by toasterparty** [comment]
> Make the appropriate changelog entries as well.

Done

**Pull request merged**